lms/envs/common.py

```diff
@@ -1633,7 +1633,8 @@
 # then you need to add the js urls in this list.
 REQUIRE_JS_PATH_OVERRIDES = [
     'js/bookmarks/views/bookmark_button.js',
-    'js/views/message_banner.js'
+    'js/views/message_banner.js',
+    'js/vendor/draggabilly.pkgd.js'
```

Comment:
@clrux This path may not be correct (it may need a prefix since draggabilly lives in common, whereas the other two files are LMS files). This is the configuration that should prevent the uncached version of the file from appearing. I recommend that you work with @muhammad-ammar to sort out why you are still seeing an uncached file.

It is expected that moment is uncached-- there is an outstanding bug to fix it (presumably in the same way that you are registering draggabilly).